### PR TITLE
Alee/cvsl 1910

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: server/routes/approvingLicences/handlers/approvalCases.test.ts [hunk 1]
```diff
@@ -4,11 +4,11 @@ import ApprovalCaseRoutes from './approvalCases'
 import PrisonerService from '../../../services/prisonerService'
 
 import type { PrisonDetail } from '../../../@types/prisonApiClientTypes'
-import ApproverCaseloadService, { type ApprovalCase } from '../../../services/approverCaseloadService'
```
Comment (andrewrlee) [inline server/routes/approvingLicences/handlers/approvalCases.test.ts:7]: a lot of this is repackaging

File: server/routes/viewingLicences/handlers/viewCases.test.ts [hunk 2]
```diff
@@ -1748,7 +1735,7 @@ describe('Route handlers - View and print case list', () => {
       })
 
       it('should return a previously approved licence', async () => {
-        const caseList = new Container([
```
Comment (andrewrlee) [inline server/routes/viewingLicences/handlers/viewCases.test.ts:1751]: no longer using this Container class in a load of places - it wasn't providing enough value for it's weight

File: server/services/lists/caselistUtils.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
+import moment from 'moment'
+import { isFuture, isWithinInterval, sub } from 'date-fns'
+import type { ManagedCase } from '../../@types/managedCase'
+import { parseIsoDate } from '../../utils/utils'
+
+export default class CaseListUtils {
+  public static isRecall(offender: ManagedCase): boolean {
```
Comment (andrewrlee) [inline server/services/lists/caselistUtils.ts:7]: Extracted out these utils which are used in a few places

**Review by nishanthmahasamudram — APPROVED** [review]
LGTM

**Review by louisenorris — APPROVED** [review]
LGTM

**Pull request merged**